docker-compose.yml

```diff
@@ -12,6 +12,18 @@ services:
     ports:
       - 5000:5000
 
+  tweet-api:
+    build:
```

Comment:
Why is this a build arg? 🤔
It looks like this should be a runtime variable at best.

Reply:
That was my bad; `FLASK_DEBUG` can be removed as a build arg